JNI/cpp/StellarPassphrase.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+//
+// This is a GENERATED FILE, changes made here WILL BE LOST.
+//
+
+#ifndef JNI_TW_STELLARPASSPHRASE_H
+#define JNI_TW_STELLARPASSPHRASE_H
+
+#include <jni.h>
+#include <TrustWalletCore/TWBase.h>
+
+TW_EXTERN_C_BEGIN
+
```

Comment:
Why empty?

Reply:
I think this is just a header file? not sure

Reply:
It's an enum with no exported methods.